.setup/install_submitty/setup_directories.sh

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env bash
+set -ve
+
+source "$( dirname "${BASH_SOURCE[0]}" )/get_globals.sh"
+
```

Comment:
should we add the check that this is run as root user at the top of this script (and all other helper install scripts)?

Reply:
Sure. I left that in `SUBMITTY_INSTALL_HELPER.sh` without thinking much of it; we could put a check for `root` in `get_globals.sh` because it gets called by everything else.